core/src/main/java/com/netflix/conductor/core/utils/JsonUtils.java

```diff
@@ -0,0 +1,68 @@
+package com.netflix.conductor.core.utils;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.google.common.annotations.VisibleForTesting;
+
+import javax.inject.Singleton;
+import java.util.List;
+import java.util.Map;
+
+@Singleton
+public class JsonUtils {
```

Comment:
Missing java doc at the class level, having javadoc at class level will help future use cases.

Reply:
Added javadoc